pagopa/io-app
# feat: [IOCOM-522] Paid payments details' screen for a cancelled PN message

⚠️ This PR depends on #4977 ⚠️

## Short description
This PR adds the detailed payment information screen for a PN paid payment on a cancelled notification.
<img width="413" alt="Screenshot 2023-09-15 at 15 29 02" src="https://github.com/pagopa/io-app/assets/5150343/e17cad50-f99c-457d-9786-6568ae0954aa">

## List of changes proposed in this pull request
- New `PnMessagePaymentScreen` to show the payment info (with all the react-navigation changes to support it)
- `PnMessagePayment` added the logic to navigate to such screen with proper data

## How to test
Make sure to have the local feature flag on for the new PN component. Run the io-dev-api-server with a PN configuration that generates a cancelled message with one valid paid payment. Navigate to such message details' screen and tap on the payment.
